### ISSUE TITLE
Rust language guide html page seems to be missing on stable-aarch64-apple-darwin builds of Rust

### ISSUE THREAD

**Issue by andylin2004** [rust-lang/rust#94528]
I tried this command on my MacBook Air M1 using the stable-aarch64-apple-darwin build of Rust:

```sh
rustup doc --book
```

I expected to see this happen: The html page to the Rust language guide opens up in my browser of choice.

Instead, this happened: 

```sh
error: couldn't open browser: command 'open' did not execute successfully; exit status: 1
command stderr:
The file /Users/andylin/.rustup/toolchains/stable-aarch64-apple-darwin/share/doc/rust/html/book/index.html does not exist.
```

**Comment by zerj9** [comment]
Looks like this is a known issue. There's a workaround here: https://github.com/rust-lang/rustup/issues/2692#issuecomment-848406797

**Comment by ehuss** [comment]
This has been fixed in 1.60 via #92800.

If you install beta or nightly, you should be able to get the rust-docs component.  This should hit stable in about 5 weeks.  Closing as fixed, however, feel free to let us know if it is not working in newer releases.
